react-native/react/get-element-size-or-position-with-onLayout.mo.md

```diff
@@ -0,0 +1,81 @@
+# [MO] Get element size or position dynamically with onLayout
+
+## Owner: [Julien Nassar](https://github.com/juliennassar)
+
+## Control Points
+
+- Hide your dynamically sized (or positioned) component width a `style={{ opacity: 0 }}` during the first rendering of your component to avoid glitches.
+
+## Motivation
+
+If you need :
+- to create a custom loading bar display
```

Comment:
Why do you actually want to use onLayout to do this? You can do without it using the a percentage width: https://snack.expo.io/@louiszawadzki/loading-bar.

I have no proof but I believe using onLayout could be slower (or at least demand more computation) than using styles.